event_handler/sources.py

```diff
@@ -84,27 +84,25 @@ def get_source(headers):
     Gets the source from the User-Agent header
     """
     if "X-Gitlab-Event" in headers:
-        return "Gitlab"
+        return "gitlab"
 
-    if "Ce-Type" in headers and "tekton" in headers["Ce-Type"]:
-        return "Tekton"
+    if "tekton" in headers.get("Ce-Type", ""):
+        return "tekton"
 
-    if "User-Agent" in headers:
-        if "/" in headers["User-Agent"]:
-            return headers["User-Agent"].split("/")[0]
-        return headers["User-Agent"]
-
-    return None
+    if "GitHub" in headers.get("User-Agent", ""):
+        return "github"
+    
+    return "ERROR: UNKNOWN EVENT SOURCE"
```

Comment:
Can we make this return the User Agent, so this raises a useful error message? https://github.com/GoogleCloudPlatform/fourkeys/blob/main/event_handler/event_handler.py#L40

Reply:
Yup done.